gluoncv/__init__.py

```diff
@@ -8,6 +8,13 @@
 from .utils.version import _require_mxnet_version, _deprecate_python2
 _deprecate_python2()
 _require_mxnet_version('1.4.0')
+try:
+    # pylint: disable=unused-import
+    from mxnet import metric
+except ImportError:
+    from mxnet.gluon import metric
```

Comment:
There are API changes besides the rename: https://github.com/apache/incubator-mxnet/issues/18046